driver/csiplugin/controllerserver.go

```diff
@@ -604,7 +609,7 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 			srcVolumeIDMembers, err = getVolIDMembers(srcVolumeID)
 			if err != nil {
 				glog.Errorf("volume:[%v] - Invalid Volume ID %s [%v]", volName, srcVolumeID, err)
-				return nil, err
+				return nil, status.Error(codes.NotFound, "volume source volume is not found")
```

Comment:
Lets keep err in return as well

Reply:
Done changes in  new PR https://github.com/IBM/ibm-spectrum-scale-csi/pull/842